src/documentation/0000-node-fs-fd-use-case-read-method/article.md

```diff
@@ -0,0 +1,58 @@
+* **Start Date:** 2021-06-28
+* **PR:** (leave this empty)
+* **Keywords:** node fs read file-descriptor
+* **Summary:** Use case of file descriptor returned by fs.open() method in fs.read()
+
+# Introduction to fs module read method.
+
+Using `fs.read()` method we can read a file in a more controlled manner in comparision to `fs.readFile()` or `fs.readFileSync()`.
+
+## Working with fs.read() method.
+
+Since `fs.readFile()` and `fs.readFileSync()` reads the full content of the file in memory before returning the data or in other words before you access and start operating on the data that is read, the methods will make sure the entire file is read which in case of huge sized file will have a great performance impact in terms of memory usage and speed of execution of the program.
+
+So, we can manage the above situation by using `fs.read()` method which give us more control over reading the file. How ? We will see it below.
```

Comment:
```suggestion
So, we can manage the above situation by using `fs.read()` method which give us more control over reading the file. How? We will see it below.
```

Reply:
@s-mrb I have committed your suggestion for both the modules.